tests/python/test_basic.py

```diff
@@ -16,32 +18,31 @@
 class TestBasic:
     def test_compat(self):
```

Comment:
mypy doesn't check the function if there's no annotation in the signature.
```suggestion
    def test_compat(self) -> None:
```